integration-cli/requirements_unix_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 	"strings"
 
+	cgroups "github.com/containerd/cgroups/v3"
```

Comment:
nit; alias is redundant here;

```suggestion
	"github.com/containerd/cgroups/v3"
```


Alternatively we could use the same alias as we use elsewhere (but in those cases we alias because we import both `containerd/cgroups` and `opencontainers/cgroups`;

```go
cdcgroups "github.com/containerd/cgroups/v3"
```